src/components/CookieWarning.js

```diff
@@ -33,27 +72,34 @@ const CookieWarning = () => {
     }, cookies)
     signOut()
   }
-  return !cookiesAccepted && aside({
+  return showWarning && aside({
     'aria-label': 'cookie consent banner',
     style: {
-      flex: 0, height: 100, width: '100%',
-      display: 'flex', alignItems: 'center', justifyContent: 'space-between',
-      backgroundColor: colors.dark(0.15), borderBottom: `6px solid ${colors.primary()}`
+      display: 'block', width: '100%',
```

Comment:
With block, if the page is really tall, does it show up on the page? I would worry that we might need to do something like `position: 'absolute'`?

Reply:
Im sorry, I don't understand the question! How does block get affected by the page length? 

I'm using `position: 'fixed'` already, which i prefer over `absolute`